terraform/modules/helm/flink/main.tf

```diff
@@ -19,8 +19,8 @@ resource "helm_release" "flink_sa" {
   ]
 }
 resource "helm_release" "flink" {
-    count            = length(var.flink_release_name)
-    name             = var.flink_release_name[count.index]
+    for_each         = var.flink_release_map
```

Comment:
@ManojCKrishna  Do you want to rename the variable to flink_job_map and also the next variable to job_name?